themes/finna2/templates/Helpers/organisation-info-contacts.phtml

```diff
@@ -1,23 +1,18 @@
-<h4 class="panel-title">
-  <a data-toggle="collapse" href="#collapse<?=$this->escapeHtmlAttr($this->id)?>"><?=$this->translate('organisation_info_contact')?></a>
-</h4>
-<div id="collapse<?=$this->escapeHtmlAttr($this->id)?>" class="panel-collapse collapse in">
-  <div class="email-contact info-element">
-    <strong><?=$this->translate('organisation_info_email')?></strong>
-    <div class="emails">
-      <p><i class="fa fa-envelope"></i> <a href="#" class="email"><span class="email"></span></a></p>
-    </div>
-  </div>
-  <div class="address-contact info-element">
-    <strong><?=$this->translate('organisation_info_address')?></strong>
-    <p></p>
-  </div>
-  <div class="mail-address-contact info-element">
-    <strong><?=$this->translate('organisation_info_address_mail')?></strong>
-    <p></p>
-  </div>
-  <div class="phone-numbers info-element">
-    <strong><span class="contact-info-header"><?=$this->translate('organisation_info_phone_numbers')?></span></strong>
-    <p></p>
+<div class="email-contact info-element">
+  <span><?=$this->transEsc('organisation_info_email')?></span>
+  <div class="emails">
+    <p><i class="fa fa-envelope"></i> <a href="#" class="email"><span class="email"></span></a></p>
   </div>
 </div>
+<div class="address-contact info-element">
+  <span><?=$this->transEsc('organisation_info_address')?></span>
+  <p></p>
```

Comment:
Tyhjät p-elementit on kyllä jo vanhassa koodissa, mutta mainitsen kuitenkin. Tarvitaanko niitä placeholdereina vai ovatko turhia... rivit 9, 13, 17?
(Voi olla, että ne tarvitaan mutta lähtökohtaisesti tyhjät elementit ilman id:tä kummastuttaa ja javascript/ajax muokkauksissakin kai usein suositaan id:itä)

Reply:
Ne ovat placeholdereita jotka täytetään JS-koodista (`finna-organisation-info-page.js`):

`address.show().find('> p').html(data.address);`

Olisi kyllä hyvä tehdä id:llä, mutta mikäli tällä komponentti-toteutustavalla edetään, tämä olisi joka tapauksessa vielä tulossa omana issuenaan.

Reply:
Ok, hyvä - asia selvä.